source/utilities.cc

```diff
@@ -556,6 +569,7 @@ namespace aspect
       MapParsing::Options options(input_field_names, property_name);
       options.allow_multiple_values_per_key = allow_multiple_values_per_key;
       options.allow_missing_keys = allow_missing_keys;
+      options.use_differences_instead_of_values = use_differences_instead_of_values;
```

Comment:
So instead of handling the values of inputs, I can only pass this to the instance of this class. This class would then go through the interface of another function and reach the point where the inputs are actually handled (you can follow it in the code).